### ISSUE TITLE
Higher Genus Curve download not working

### ISSUE THREAD

**Issue by jenpaulhus** [LMFDB/lmfdb#1663]
The pages from #1657 are returning an error in beta.
http://beta.lmfdb.org/HigherGenus/C/aut/3.14-2.0.2-7-14/download/magma

When I checked locally, it looks like the issue is that "make_response" is not being imported from flask which is called on line 497 of main.py.

However, when I look at the files I committed (which include the import), they are a bit different than the ones that were merged
https://github.com/LMFDB/lmfdb/pull/1657/commits/1c1ff28b47fa2e2520b0694e28a8f437035e095f
vs
https://github.com/LMFDB/lmfdb/commit/c2d10f19d6d652279f2099a3be6f8bc5bec5959a

Somehow along the way a few imported commands got lost.  Is there a way to fix this directly or should I just make a new pull request with that command added back in?

**Comment by AndrewVSutherland2** [comment]
@jenpaulhus -- let me figure out what went wrong here before you make any
further changes.

On Thu, Jul 7, 2016 at 8:23 AM, Jen Paulhus, Ph.D. <notifications@github.com

**Comment by AndrewVSutherland** [comment]
@jenpaulhus OK, the make_response import inadvertently got removed in PR #1592 (which was applied to a version of main.py that was not using make_response).  This wasn't flagged as a merge conflict by git because your most recent PR didn't change this line.  I have corrected it and pushed the change to beta, once I confirm the fix works I will close this issue.

pyflakes did not report any other missing imports, so this should be the only issue.  But it did notice that line line 542 of main.py the variable num_entries is assigned but never used -- you might want to take a look at this.
